picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +436,29 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+
+    /**
+     * Set a list of transformations to be applied to the image.  These transformations will
+     * overwrite any existing transformations.
```

Comment:
I thought it was a bit odd that you can dump these transformations into the `RequestCreator` but you can't remove them or access them afterwards.  This approach allows you to take an existing request creator and remove any transformations.